vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -98,7 +98,7 @@ public ExperimentDetailsComponent(@Autowired ProjectInformationService projectIn
   }
 
   public DisclaimerCard createDisclaimer() {
-    var disclaimer = DisclaimerCard.create("No experiment variables defined", "Add");
+    var disclaimer = DisclaimerCard.create("No experiment variables defined", "Add", "Missing variables");
```

Comment:
NIT: Three string parameters in a weird order (title last) could get confusing. I think a factory would be overkill, though?

Reply:
alternatively I can create more explicit creation methods. BTW: these are factory creation methods :D

Reply:
okay, fair enough :)